app/models/search_settings.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: search_settings
+#
+#  id           :integer          not null, primary key
+#  community_id :integer          not null
+#  main_search  :string(255)      default("KEYWORD"), not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+
+class SearchSettings < ActiveRecord::Base
+  attr_accessible(
```

Comment:
@sktoiva Good catch, that's exactly right. We shouldn't use `attr_accessible` anymore. I'd also consider about the presence validators. Is it a Model level thing or should that be also done in Controller/Service API ?